src/react-components/src/components/01-atoms/Button/Button.tsx

```diff
@@ -37,12 +66,12 @@ export default class Button extends React.Component<ButtonProps, {}> {
   }
 
   render(): JSX.Element {
-    const { id, callback, content, attributes, modifiers, blockName, type, mouseDown,
+    const { id, callback, content, attributes, modifiers, blockName, type, buttonType = "submit", mouseDown,
```

Comment:
I thought we were no longer having `submit` as an assumption on the buttons?

Reply:
this assigns it if it's empty, since it's optional.  We could make buttonType required instead, though.

Reply:
I think it would be safe to make the assumption that if `buttonType` wasn't passed, it should default to primary.